### ISSUE TITLE
How to parse date with a given format?

### ISSUE THREAD

**Issue by sohailalam2** [iamkun/dayjs#289]
This works pretty well in momentjs 

```js
moment('27/08/2018', 'DD/MM/YYYY')
```

How can we make it work in dayjs? 

This fails with the following error message - 

```js
dayjs('27/08/2018', 'DD/MM/YYYY')
```

```
TypeError: Cannot create property 'date' on string 'DD/MM/YYYY'
```

**Comment by kyleoliveiro** [comment]
Any update on this? Pretty essential feature for people outside the US.

**Comment by subpx** [comment]
Agree that this is needed. Do a lot of work with dates and would not move from moment without this feature. Lib looks nice.

**Comment by iamkun** [comment]
:tada: CustomParseFormat plugin is supported and included in version 1.8.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/dayjs)
- [GitHub release](https://github.com/iamkun/dayjs/releases/tag/v1.8.0)
